src/Resources/config/doctrine/aggregator.mongodb.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<doctrine-mongo-mapping xmlns="http://doctrine-project.org/schemas/odm/doctrine-mongo-mapping"
+                        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                        xsi:schemaLocation="http://doctrine-project.org/schemas/odm/doctrine-mongo-mapping
+                        http://doctrine-project.org/schemas/odm/doctrine-mongo-mapping.xsd">
+
+    <mapped-superclass name="Algolia\SearchBundle\Entity\Aggregator">
```

Comment:
In ODM 1.x, identifiers are mapped like this:

```xml
<field name="id" id="true" />
```

While this will store your typical BSON ObjectID in the database, it is exposed as a string in the PHP object, so there are no changes necessary apart from the mapping.

The identifier mapping changes in ODM 2.0. To avoid future issues, you should already ensure that these mappings can only be used with ODM 1.0. Currently you can do that by adding a `conflict` section to composer.json:
```json
"conflict": {
    "doctrine/mongodb-odm": ">= 2.0"
}
```

I'm working on a solution to allow for bundles providing mappings for ODM 1.x and 2.x at the same time and will contribute this solution to this bundle once I've got it figured out 👍

Reply:
Fixed.